modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -42,7 +43,7 @@ spec:
 <3> The name for the configuration. It is recommended to match the configuration name to the `name` value of the network attachment definition.
 <4> The actual name of the Container Network Interface (CNI) plug-in that provides the network for this network attachment definition. Do not change this field unless you want to use a different CNI.
 <5> The name of the Linux bridge configured on the node.
-<6> Optional: Flag to enable MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.
+<6> Optional: Flag to enable MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod. By default, this is set to `true` for {VirtProductName} 4.9 and later. For earlier versions of {VirtProductName}, explicitly set the attribute, then restart or migrate the VM.
```

Comment:
> By default, this is set to true for OpenShift Virtualization 4.9 and later. For earlier versions of OpenShift Virtualization, explicitly set the attribute, then restart or migrate the VM.

I guess /me not in the loop here.
What default are you referring to? The example explicitly setting it to `true`?

The [upstream CNI](https://www.cni.dev/plugins/current/main/bridge/) is explicitly saying it is by default `false`, so I am not sure who or what sets a different default.

Reply:
LGTM